src/app/components/FatalErrorHandler/index.tsx

```diff
@@ -82,40 +76,38 @@ export function FatalErrorHandler({ children }: Props) {
           />
         </Text>
         <Box direction={isMobile ? 'column' : 'row'} margin={{ bottom: isMobile ? 'xlarge' : 'large' }}>
-          <StyledButton
+          <BrandButton
             href="https://status.oasisprotocol.org"
             target="_blank"
             rel="noopener noreferrer"
-            icon={<Dashboard size="22px" color={normalizeColor('brand-white', theme)} />}
+            icon={<Dashboard />}
             label={t('fatalError.checkStatus', 'Check network status')}
-            style={{
-              backgroundColor: normalizeColor('brand-blue', theme),
-              color: normalizeColor('brand-white', theme),
-              borderColor: normalizeColor('brand-blue', theme),
-            }}
-            size="large"
           />
         </Box>
         <Box direction="row" justify="end" margin={{ bottom: isMobile ? 'medium' : 'small' }}>
           <StyledTextArea data-testid="fatalerror-stacktrace" readOnly rows={6} value={combinedStacktrace} />
         </Box>
         <Box align={isMobile ? 'stretch' : 'end'} margin={{ bottom: isMobile ? 'xlarge' : 'large' }}>
-          <StyledButton
+          <BrandButton
+            brandVariant="gray"
             onClick={() => copyError()}
-            icon={<Copy size="18px" color={normalizeColor('brand-gray-extra-dark', theme)} />}
+            icon={<Copy />}
             label={
               !copied
                 ? t('fatalError.copy', 'Copy error to clipboard')
                 : t('fatalError.copied', 'Error copied to clipboard')
             }
-            style={{
-              backgroundColor: normalizeColor('brand-gray-medium', theme),
-              color: normalizeColor('brand-gray-extra-dark', theme),
-              borderColor: normalizeColor('brand-gray-medium', theme),
-            }}
-            size="large"
           />
         </Box>
+        {runtimeIs === 'extension' && (
+          <Box align={isMobile ? 'stretch' : 'end'} margin={{ bottom: isMobile ? 'xlarge' : 'large' }}>
```

Comment:
preview
![image](https://github.com/oasisprotocol/oasis-wallet-web/assets/3758846/b1b34c6d-3c19-49c5-8f33-9f3f1c94d6b4)

Reply:
will ask Don tomorrow if we want to move this button up

Reply:
![Screenshot from 2024-04-02 11-28-37](https://github.com/oasisprotocol/oasis-wallet-web/assets/891392/6f26695a-d459-485b-840e-e82f284d4881)